modulles/news.modules.js

```diff
@@ -66,3 +66,12 @@ exports.postCommentToDb = (comment) => {
     "INSERT INTO comments (body, author, article_id, votes, created_at) VALUES %L  RETURNING *;";
   return db.query(format(queryStr, [comment]));
 };
+
+exports.deleteFromDb = ({ table_name, column, id }) => {
+  const queryPsql = `DELETE FROM ${table_name} WHERE ${column} = $1 RETURNING*;`;
```

Comment:
The problem with using template literals here is it leaves you open for psql injection. If you want this re-usable function, you'll need to sanitise and check the table name and column before you use it in the string

Reply:
It would be easier just to make a specific model to delete from this table and hardcode in the values